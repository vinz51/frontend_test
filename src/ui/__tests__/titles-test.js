import TitleBold, { SubTitleBold, H1, Bold } from "../titles";
import renderer from "react-test-renderer";
import React from "react";

describe("Titles", () => {
  describe("Snapshots", () => {
    it.each([
      ["title in bold", TitleBold],
      ["subtitle in bold", SubTitleBold],
      ["bold", Bold],
      ["h1", H1],
    ])("renders %s", (title, Component) => {
      const myTreeComponent = renderer
        .create(<Component>Title</Component>)
        .toJSON();

      expect(myTreeComponent).toMatchSnapshot();
    });
  });
});
