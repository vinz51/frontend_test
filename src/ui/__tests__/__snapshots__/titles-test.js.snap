// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Titles Snapshots renders bold 1`] = `
<h2
  className="sc-AxirZ kDwUrB"
>
  Title
</h2>
`;

exports[`Titles Snapshots renders h1 1`] = `
<h1
  className="sc-AxjAm euwmKR"
>
  Title
</h1>
`;

exports[`Titles Snapshots renders subtitle in bold 1`] = `
<h2
  className="sc-AxirZ sc-AxhCb dspALo"
>
  Title
</h2>
`;

exports[`Titles Snapshots renders title in bold 1`] = `
<h2
  className="sc-AxirZ sc-AxiKw hYWAVI"
>
  Title
</h2>
`;
